src/Shared/UnitTests/ObjectModelHelpers.cs

```diff
@@ -725,7 +746,7 @@ internal static void BuildProjectExpectSuccess
             params ILogger[] loggers
             )
         {
-            Project project = CreateInMemoryProject(projectContents, logger: null); // logger is null so we take care of loggers ourselves
```

Comment:
Since we were explicitly nulling out the logger here, we were dropping all evaluation events.